monai/networks/trt_wrapper.py

```diff
@@ -0,0 +1,440 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import inspect
+import os
+import threading
+from collections import OrderedDict
+
+import torch
+
+from monai.apps.utils import get_logger
+from monai.networks.utils import add_casts_around_norms, convert_to_onnx
+from monai.utils.module import optional_import
+
+P, P_imported = optional_import("polygraphy")
+if P_imported:
+    from polygraphy.backend.common import bytes_from_path
+    from polygraphy.backend.trt import (
+        CreateConfig,
+        Profile,
+        engine_from_bytes,
+        engine_from_network,
+        network_from_onnx_path,
+        save_engine,
+    )
+
+trt, trt_imported = optional_import("tensorrt")
+cudart, _ = optional_import("cuda.cudart")
+
+LOGGER = get_logger("run_cmd")
```

Comment:
The name of the logger should be something about TRT wrapper maybe?

Reply:
Done.